src/pages/userManagement/UserManagerCollaborationEditTable.jsx

```diff
@@ -170,7 +185,7 @@ export default function UserManagersCollaborationEditTable({
           style={{ margin: '8px 16px', display: 'block' }}
         />
       ) : null}
-      {isError && !dismissed ? (
+      {patchError && !dismissed ? (
```

Comment:
why keep track of dismissed instead of using the `onClearError` functionality we have built?

Reply:
This code pre-dated our onClearError pattern. Fixed now.